lib/elixir_conf_africa_web/live/home_live/index.ex

```diff
@@ -0,0 +1,14 @@
+defmodule ElixirConfAfricaWeb.HomeLive.Index do
+  use ElixirConfAfricaWeb, :live_view
+  alias ElixirConfAfrica.Events
+
+  def mount(_params, _session, socket) do
+    elixir_conf_africa_event = Events.get_elixir_conf_event_and_ticket_types()
+    availabe_tickets = 20
```

Comment:
We should avoid static values

Reply:
I have now set it up such that it takes  the available tickets